walkthroughs/nbconverted/single_cell_usage.py

```diff
@@ -0,0 +1,248 @@
+# compressed csv file
+sc_profile = single_cell_profile.merge_single_cells(
+    sc_output_file=sc_profiles_path, compression_options="gzip"
+)
+
+
+# Now that we have created our merged single-cell profile, let's move on to the next step: loading our `platemaps`. 
+# 
+# Platemaps provide us with additional information that is crucial for our analysis. They contain details such as well positions, genotypes, gene names, perturbation types, and more. In other words, platemaps serve as a valuable source of metadata for our single-cell morphology profile.
+# 
+
+# In[5]:
+
+
+# loading plate map and display it
+platemap_df = pd.read_csv(plate_map)
+
+# displaying platemap
+print(platemap_df.columns.tolist())
+
+
+# ## Annotation
+# 
+# In this step of the walkthrough, we will combine the metadata with the merged single-cell morphology dataset. To accomplish this, we will utilize the `annotation` function provided by `pycytominer`.
+# 
+# The `annotation` function takes two inputs: the merged single-cell morphology dataset and its associated plate map. By combining these two datasets, we will generate an annotated_profile that contains enriched information.
+# 
+# More information about the `annotation` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.annotate)
+# 
+
+# In[6]:
+
+
+# annotating merged single-cell profile with metadata
+annotate(
+    profiles=sc_profile,
+    platemap=platemap_df,
+    join_on=["Metadata_well_position", "Image_Metadata_Well"],
+    output_file=anno_profiles_path,
+    compression_options="gzip",
+)
+
+# save message display
+print(f"Annotated profile saved in: {anno_profiles_path}")
+
+
+# ## Normalization Step
+# 
+# The next step is to normalize our dataset using the `normalize` function provided by `pycytominer`.
+# More information regards `pycytominer`'s `normalize` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.normalize)
+# 
+# Normalization is a critical preprocessing step that improves the quality of our dataset. It addresses two key challenges: mitigating the impact of outliers and handling variations in value scales. By normalizing the data, we ensure that our downstream analysis is not heavily influenced by these factors.
+# 
+# Additionally, normalization plays a crucial role in determining feature importance (which is crucial for our last step). By bringing all features to a comparable scale, it enables the identification of important features without biases caused by outliers or widely-scaled values.
+# 
+# To normalize our annotated single-cell morphology profile, we will utilize the `normalize` function from `pycytominer`. This function is specifically designed to handle the normalization process for cytometry data. 
+
+# In[7]:
+
+
+# load annotated profile
+annotated_df = load_profiles(anno_profiles_path)
+
+# normalize dataset
+normalize(
+    profiles=annotated_df,
```

Comment:
normalize (and all other pycytominer functions) also work if you provide a path.

So, there is no need to use `load_profiles()` (also no need to import load_profiles)

We should stick to using the path convention